### ISSUE TITLE
Top PCB has only 0603 components

### ISSUE THREAD

**Issue by jeanthom** [jeanthom/Otter-Iron-PRO-BOM#1]
This mean we can't reuse the 0402 parts (2k37 resistors and 100n capacitors) from the bottom PCB.

**Comment by jeanthom** [comment]
2k37 0603: NRC-AS06F2371TRF
100n 0603 X7R 50V: C0603C104K5RAC9109

**Comment by Jana-Marie** [comment]
In retrospect it would have been better to use 0402 ^^

**Comment by jeanthom** [comment]
Yep that would have reduced the amount of lines the BOM, but it's not that bad since you can solder 0402s in a 0603 footprint without much trouble.
